__tests__/formats/__snapshots__/typeScriptEs6Declarations.test.snap.js

```diff
@@ -39,9 +42,21 @@ snapshots["formats typescript/es6-declarations with DTCG tokens and without outp
  * Do not edit directly, this file was auto-generated.
  */
 
+export const colorBlue: string;
 /** Used for errors */
 export const colorRed: string;
 export const fontFamily: string;
 `;
 /* end snapshot formats typescript/es6-declarations with DTCG tokens and without outputStringLiterals should match snapshot */
 
+snapshots["formats typescript/es6-declarations with formatted comment should match snapshot"] = 
+`/**
+ * Do not edit directly, this file was auto-generated.
+ */
+
+export const colorBlue: string; /** Used for errors */
+export const colorRed: string;
```

Comment:
```suggestion
export const colorBlue: string;
/** Used for errors */
export const colorRed: string;
```

this would be my expectation, the comment seems to be attached to the red token, not the blue, and in the format it says long / above are enforced to match typescript declaration emission behavior.

The code looks okay, maybe this snapshot just needs to be updated with `npm run test:update-snapshots`